projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -322,14 +322,15 @@ private function prepare_menu_item_url( $url, $parent_slug = '' ) {
 
 	/**
 	 * Parses the update count from a given menu item title and removes the associated markup.
+	 * Also remove unexpected markup from the title.
 	 *
 	 * "Plugin" and "Updates" menu items have a count badge when there are updates available.
 	 * This method parses that information and adds it to the response.
 	 *
 	 * @param string $title Title to parse.
 	 * @return array
 	 */
-	private function parse_update_count( $title ) {
+	private function parse_markup_data( $title ) {
 		$item = array();
 
 		if ( false !== strpos( $title, 'count-' ) ) {
```

Comment:
As the code block that handles count is now a side effect of this method should we consider moving it into a dedicated method called `parse_update_count` which we invoke from the revised `parse_markup_data`?